rpc/types/src/misc/status.rs

```diff
@@ -86,17 +86,12 @@ pub enum Status {
     #[cfg_attr(feature = "serde", serde(rename = "PAYMENT REQUIRED"))]
     PaymentRequired,
 
-    /// Some unknown other string; [`CORE_RPC_STATUS_UNKNOWN`].
+    /// Some unknown other string.
     ///
-    /// This exists to act as a catch-all if `monerod` adds
-    /// a string and a Cuprate node hasn't updated yet.
-    ///
-    /// The reason this isn't `Unknown(String)` is because that
-    /// disallows [`Status`] to be [`Copy`], and thus other types
-    /// that contain it.
-    #[cfg_attr(feature = "serde", serde(other))]
-    #[cfg_attr(feature = "serde", serde(rename = "UNKNOWN"))]
-    Unknown,
+    /// This exists to act as a catch-all for all of
+    /// `monerod`'s other strings it puts in the `status` field.
+    #[cfg_attr(feature = "serde", serde(rename = "OTHER"), serde(untagged))]
+    Other(String),
```

Comment:
`enum Status` now has a catch-all and is no longer `Copy`.

Turns out `monerod` emits a lot of `status` strings other than the ones listed...